### ISSUE TITLE
can't click github link due to language changer

### ISSUE THREAD

**Issue by muelli** [openSUSE/osem#1056]
It's very hard me for to hit the "github" link. 
![nogithubutton](https://cloud.githubusercontent.com/assets/322807/16377191/e4ced804-3c64-11e6-8a6a-58af79a9eaac.png)

See the screenshot.

**Comment by differentreality** [comment]
Yeah, we should probably show the language selector box above the footer. 
Not sure how to do this with CSS, in a responsive way.
